src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/diagnostic_settings/_create.py

```diff
@@ -15,7 +15,7 @@
     "monitor diagnostic-settings create",
 )
 class Create(AAZCommand):
-    """Create diagnostic settings for the specified resource. For more information, visit: https://docs.microsoft.com/rest/api/monitor/diagnosticsettings/createorupdate#metricsettings.
+    """Create diagnostic settings for the specified resource. For more information, visit: https://learn.microsoft.com/rest/api/monitor/diagnosticsettings/createorupdate#metricsettings.
```

Comment:
There is no need to edit files under `aaz` folders as they are automatically generated from swagger https://github.com/Azure/azure-rest-api-specs. They will be overwritten next time we regenerate these files.